hls4ml/model/attributes.py

```diff
@@ -15,6 +16,14 @@ def validate_value(self, value):
         else:
             return True # Meaning we don't care
 
+    @property
+    def config_name(self):
+        return convert_to_pascal_case(self.name)
+
+class ConfigurableAttribute(Attribute):
```

Comment:
A doc-string would be useful to describe configurable attributes (and maybe the case conventions?)